packages/xo-server/src/xapi-stats.js

```diff
@@ -244,7 +229,38 @@ const STATS = {
 }
 
 // -------------------------------------------------------------------
-
+// RRD
+// json: {
+//   meta: {
+//     start: Number,
+//     step: Number,
+//     end: Number,
+//     rows: Number,
+//     columns: Number,
+//     legend: String []
```

Comment:
Do we know how many items in `legend`?

Reply:
Yes, It's rows.

Reply:
Then add it to the comment :slightly_smiling_face: 

Something like: `legend: String[row]`